db/migrate/20230426150135_add_organisations_for_existing_users.rb

```diff
@@ -1,23 +1,27 @@
 class AddOrganisationsForExistingUsers < ActiveRecord::Migration[7.0]
   def up
-    Organisation.create!(content_id: "af07d5a5-df63-4ddc-9383-6a666845ebe9", slug: "government-digital-service", name: "Government Digital Service")
+    PaperTrail.request(enabled: false) do
```

Comment:
Changing an existing migration feels like one of those things you should never do! Are there any alternatives here though?

The issue is using the model within the migration. If the model relies on tables which have only been added in later migrations, here versions added by paper_trail, the migration will fail.

Reply:
This has been solved in another commit, by deleting the migration.